sdk/compiler/damlc/tests/daml-test-files/LfStableManyMoreFields.daml

```diff
@@ -0,0 +1,129 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Please do not remove the commented lines, there are helpful to
```

Comment:
Could we add a comment explaining
- what this is testing in the Engine (support for large records)
- what has changed recently that causes the test to be necessary (interning)
- the reasoning behind using 600
- why 120 is too low of a limit?